2022-11-24 02:24:30,647 loglevel=WARNING logger=services:get_top_search_names_and_probabilities() L92   Single entry upload failed.

QUERY
:{'first_name': 'James', 'last_name': 'Jones', 'identification': '000000000', 'birthdate': '2000-01-01'}
2022-11-24 02:24:30,650 loglevel=WARNING logger=services:get_top_search_names_and_probabilities() L92   Single entry upload failed.

QUERY
:{'first_name': 'James', 'last_name': 'Jon', 'identification': '000000000', 'birthdate': '2000-01-01'}
2022-11-24 02:24:35,717 loglevel=ERROR  logger=endpoints:search_user_in_es() L32   Connection error caused by: ConnectionError(Connection error caused by: ClientConnectorError(Cannot connect to host elasticsearch:9200 ssl:default [Temporary failure in name resolution]))
2022-11-24 02:24:35,724 loglevel=ERROR  logger=services:update_single_entry() L273  

Single entry upload failed.

Event loop is closed

Traceback (most recent call last):
  File "/home/caleb/Desktop/dev_projects/document_search_app/backend/app/services/elastic_search.py", line 271, in update_single_entry
    return await self.session.index(index=index, body=input_json)
  File "/home/caleb/Desktop/dev_projects/document_search_app/backend/.venv/lib/python3.9/site-packages/elasticsearch/_async/client/__init__.py", line 2267, in index
    return await self.perform_request(  # type: ignore[return-value]
  File "/home/caleb/Desktop/dev_projects/document_search_app/backend/.venv/lib/python3.9/site-packages/elasticsearch/_async/client/_base.py", line 286, in perform_request
    meta, resp_body = await self.transport.perform_request(
  File "/home/caleb/Desktop/dev_projects/document_search_app/backend/.venv/lib/python3.9/site-packages/elasticapm/instrumentation/packages/asyncio/elasticsearch.py", line 105, in call
    result_data = await wrapped(*args, **kwargs)
  File "/home/caleb/Desktop/dev_projects/document_search_app/backend/.venv/lib/python3.9/site-packages/elastic_transport/_async_transport.py", line 259, in perform_request
    resp = await node.perform_request(
  File "/home/caleb/Desktop/dev_projects/document_search_app/backend/.venv/lib/python3.9/site-packages/elasticapm/instrumentation/packages/asyncio/elasticsearch.py", line 63, in call
    result = await wrapped(*args, **kwargs)
  File "/home/caleb/Desktop/dev_projects/document_search_app/backend/.venv/lib/python3.9/site-packages/elastic_transport/_node/_http_aiohttp.py", line 179, in perform_request
    async with self.session.request(
  File "/home/caleb/Desktop/dev_projects/document_search_app/backend/.venv/lib/python3.9/site-packages/aiohttp/client.py", line 1141, in __aenter__
    self._resp = await self._coro
  File "/home/caleb/Desktop/dev_projects/document_search_app/backend/.venv/lib/python3.9/site-packages/elasticapm/instrumentation/packages/asyncio/aiohttp_client.py", line 71, in call
    response = await wrapped(*args, **kwargs)
  File "/home/caleb/Desktop/dev_projects/document_search_app/backend/.venv/lib/python3.9/site-packages/aiohttp/client.py", line 448, in _request
    handle = tm.start()
  File "/home/caleb/Desktop/dev_projects/document_search_app/backend/.venv/lib/python3.9/site-packages/aiohttp/helpers.py", line 651, in start
    return self._loop.call_at(when, self.__call__)
  File "/home/caleb/miniconda3/lib/python3.9/asyncio/base_events.py", line 730, in call_at
    self._check_closed()
  File "/home/caleb/miniconda3/lib/python3.9/asyncio/base_events.py", line 515, in _check_closed
    raise RuntimeError('Event loop is closed')
RuntimeError: Event loop is closed
2022-11-24 02:24:35,729 loglevel=ERROR  logger=services:update_bulk_entries() L297  

Single entry upload failed.

Event loop is closed

Traceback (most recent call last):
  File "/home/caleb/Desktop/dev_projects/document_search_app/backend/app/services/elastic_search.py", line 293, in update_bulk_entries
    await helpers.async_bulk(self.session,
  File "/home/caleb/Desktop/dev_projects/document_search_app/backend/.venv/lib/python3.9/site-packages/elasticsearch/_async/helpers.py", line 340, in async_bulk
    async for ok, item in async_streaming_bulk(
  File "/home/caleb/Desktop/dev_projects/document_search_app/backend/.venv/lib/python3.9/site-packages/elasticsearch/_async/helpers.py", line 254, in async_streaming_bulk
    async for data, (ok, info) in azip(  # type: ignore
  File "/home/caleb/Desktop/dev_projects/document_search_app/backend/.venv/lib/python3.9/site-packages/elasticsearch/_async/helpers.py", line 156, in azip
    yield tuple([await x.__anext__() for x in aiters])
  File "/home/caleb/Desktop/dev_projects/document_search_app/backend/.venv/lib/python3.9/site-packages/elasticsearch/_async/helpers.py", line 156, in <listcomp>
    yield tuple([await x.__anext__() for x in aiters])
  File "/home/caleb/Desktop/dev_projects/document_search_app/backend/.venv/lib/python3.9/site-packages/elasticsearch/_async/helpers.py", line 111, in _process_bulk_chunk
    resp = await client.bulk(*args, operations=bulk_actions, **kwargs)  # type: ignore[arg-type]
  File "/home/caleb/Desktop/dev_projects/document_search_app/backend/.venv/lib/python3.9/site-packages/elasticsearch/_async/client/__init__.py", line 705, in bulk
    return await self.perform_request(  # type: ignore[return-value]
  File "/home/caleb/Desktop/dev_projects/document_search_app/backend/.venv/lib/python3.9/site-packages/elasticsearch/_async/client/_base.py", line 286, in perform_request
    meta, resp_body = await self.transport.perform_request(
  File "/home/caleb/Desktop/dev_projects/document_search_app/backend/.venv/lib/python3.9/site-packages/elasticapm/instrumentation/packages/asyncio/elasticsearch.py", line 105, in call
    result_data = await wrapped(*args, **kwargs)
  File "/home/caleb/Desktop/dev_projects/document_search_app/backend/.venv/lib/python3.9/site-packages/elastic_transport/_async_transport.py", line 259, in perform_request
    resp = await node.perform_request(
  File "/home/caleb/Desktop/dev_projects/document_search_app/backend/.venv/lib/python3.9/site-packages/elasticapm/instrumentation/packages/asyncio/elasticsearch.py", line 63, in call
    result = await wrapped(*args, **kwargs)
  File "/home/caleb/Desktop/dev_projects/document_search_app/backend/.venv/lib/python3.9/site-packages/elastic_transport/_node/_http_aiohttp.py", line 179, in perform_request
    async with self.session.request(
  File "/home/caleb/Desktop/dev_projects/document_search_app/backend/.venv/lib/python3.9/site-packages/aiohttp/client.py", line 1141, in __aenter__
    self._resp = await self._coro
  File "/home/caleb/Desktop/dev_projects/document_search_app/backend/.venv/lib/python3.9/site-packages/elasticapm/instrumentation/packages/asyncio/aiohttp_client.py", line 71, in call
    response = await wrapped(*args, **kwargs)
  File "/home/caleb/Desktop/dev_projects/document_search_app/backend/.venv/lib/python3.9/site-packages/aiohttp/client.py", line 448, in _request
    handle = tm.start()
  File "/home/caleb/Desktop/dev_projects/document_search_app/backend/.venv/lib/python3.9/site-packages/aiohttp/helpers.py", line 651, in start
    return self._loop.call_at(when, self.__call__)
  File "/home/caleb/miniconda3/lib/python3.9/asyncio/base_events.py", line 730, in call_at
    self._check_closed()
  File "/home/caleb/miniconda3/lib/python3.9/asyncio/base_events.py", line 515, in _check_closed
    raise RuntimeError('Event loop is closed')
RuntimeError: Event loop is closed
2022-11-24 02:24:35,737 loglevel=WARNING logger=services:get_top_search_names_and_probabilities() L92   Single entry upload failed.

QUERY
:{'first_name': 'James', 'last_name': 'Jones', 'identification': '000000000', 'birthdate': '2000-01-01'}
2022-11-24 02:24:35,743 loglevel=WARNING logger=services:get_top_search_names_and_probabilities() L92   Single entry upload failed.

QUERY
:{'first_name': 'James', 'last_name': 'Jon', 'identification': '000000000', 'birthdate': '2000-01-01'}
2022-11-24 02:24:35,760 loglevel=WARNING logger=services:get_top_search_names_and_probabilities() L92   Single entry upload failed.

QUERY
:{'first_name': 'James', 'last_name': 'Jones', 'identification': '000000000', 'birthdate': '2000-01-01'}
2022-11-24 02:24:35,762 loglevel=WARNING logger=services:get_top_search_names_and_probabilities() L92   Single entry upload failed.

QUERY
:{'first_name': 'James', 'last_name': 'Jon', 'identification': '000000000', 'birthdate': '2000-01-01'}
2022-11-24 02:24:35,781 loglevel=WARNING logger=services:get_top_search_names_and_probabilities() L92   Single entry upload failed.

QUERY
:{'first_name': 'James', 'last_name': 'Jones', 'identification': '000000000', 'birthdate': '2000-01-01'}
2022-11-24 02:24:35,784 loglevel=WARNING logger=services:get_top_search_names_and_probabilities() L92   Single entry upload failed.

QUERY
:{'first_name': 'James', 'last_name': 'Jon', 'identification': '000000000', 'birthdate': '2000-01-01'}
